intel_extension_for_transformers/neural_chat/examples/quick_start/chatbot/README.md

```diff
@@ -0,0 +1,160 @@
+
+# Build Your Chatbot with Intel® Extension for Transformers neural-chat
+
+# 1. Setup Environment
+
+## 1.1 Install intel-extension-for-transformers
+
+```
+conda create -n itrex-chatbot python=3.9
+conda activate itrex-chatbot
+pip install intel-extension-for-transformers==1.3.2
+```
+## 1.2 Install neural-chat dependency
+
+```
+pip install accelerate
+pip install transformers_stream_generator
+
+git clone https://github.com/intel/intel-extension-for-transformers.git ~/itrex
+cd ~/itrex
+git checkout v1.3.2
+
+cd ~/itrex/intel_extension_for_transformers/neural_chat
+```
+
+To setup CPU platform go to 1.2.1
+To setup GPU platform go to 1.2.2
+
+### 1.2.1 CPU Platform
+`pip install -r requirements_cpu.txt`
+
+Got to 1.3
```

Comment:
1.3 or 1.3.2 ? would you please explain more for "Got to 1.3"?

Reply:
Sorry it's an error. Sections are renamed.
Fixed. Changed to "Go to secton 2".